app/src/androidTest/java/com/twilio/video/app/util/TestUtil.kt

```diff
@@ -1,27 +1,31 @@
-package com.twilio.video.app
+package com.twilio.video.app.util
 
 import android.content.Context
 import androidx.annotation.IdRes
 import androidx.test.espresso.NoMatchingViewException
 import androidx.test.platform.app.InstrumentationRegistry
 import com.google.gson.Gson
 import com.google.gson.stream.JsonReader
+import com.twilio.video.app.EmailCredentials
+import com.twilio.video.app.TestCredentials
+import junit.framework.AssertionFailedError
 import java.io.InputStreamReader
 import java.util.concurrent.TimeoutException
 
-fun retryViewMatcher(timeoutInSeconds: Long = 60000L, espressoAction: () -> Unit) {
+fun retryEspressoAction(timeoutInSeconds: Long = 60000L, espressoAction: () -> Unit) {
     val startTime = System.currentTimeMillis()
     var currentTime = 0L
     while (currentTime <= timeoutInSeconds) {
         try {
             espressoAction()
             return
         } catch (e: NoMatchingViewException) {
-            currentTime = System.currentTimeMillis() - startTime
-            Thread.sleep(10)
+            currentTime = countDown(startTime)
```

Comment:
Where is currentTime actually used?

Reply:
It's used on line 18 to check if the current time has not surpassed the timeout. If it surpasses the timeout, it throws a TimeoutException.